### PR TITLE
Updated voice activation command

### PR DESCRIPTION
**Pull request by HarryRajput**
# Summary
Changed the commands.conf file to use the word "mascot" as the activation keyword for the robot.

# Test Plan
Test by running the bot and saying "mascot".

# Test Data
N/A

### CODE REVIEW

File: PiAUISuite/VoiceCommand/commands.conf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #This is the default config file
 #These are the special options you can set (remove the #)
 #!verify==1
-!keyword==mascot
```
Comment (gammaseeker) [inline PiAUISuite/VoiceCommand/commands.conf:4]: hm. It looks like Joice already did your task for you, in that case I'm going to close this PR and just close your issue. I'll try to find another low priority task that's of interest and can teach you something.

File: PiAUISuite/VoiceCommand/commands.conf [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+!keyword== mascot
```
Comment (gammaseeker) [inline PiAUISuite/VoiceCommand/commands.conf:4]: I don't think putting spaces between the key-value pair is legal syntax for a `conf` file. Should be
```
key==value
```

**Comment by gammaseeker** [comment]
Closing this PR, Joice preemptively did this task already.

File: PiAUISuite/VoiceCommand/commands.conf (hunk 1 above)
Comment (HarryRajput) [reply PiAUISuite/VoiceCommand/commands.conf:4]: Oh okay thanks.

File: PiAUISuite/VoiceCommand/commands.conf (hunk 2 above)
Comment (HarryRajput) [reply PiAUISuite/VoiceCommand/commands.conf:4]: Got it

**Pull request closed without merging**